src/containers/Hello.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import FirstAction from '../actions/FirstAction';
+
+export class Hello extends Component {
+  componentDidMount() {
+    const { test } = this.props;
+    test();
+  }
+
+  render() {
+    return (
+      <div>
+        <h1>wassap</h1>
+      </div>
+    );
+  }
+}
+
+export const mapStateToProps = state => ({
+  index: state.index,
+});
+export default connect(
+  mapStateToProps,
+  { test: FirstAction },
+)(Hello);
```

Comment:
Ensure to have `propTypes` for your components/containers e.g. have a `test` in your `propTypes`

Reply:
i have implemented this feedback